api/v1alpha08/kogitoserverlessplatform_types.go

```diff
@@ -114,6 +160,9 @@ type PlatformConditionType string
 
 // PlatformCondition describes the state of a resource at a certain point.
 type PlatformCondition struct {
+	// TODO: the Type can't be Kubernetes or OpenShift, but the actual condition like "Ready". See the Conditions implementation in the workflow.
```

Comment:
Should we open a Jira for this?

Reply:
https://issues.redhat.com/browse/KOGITO-9218